install/debian-installation-guide.md

```diff
@@ -0,0 +1,47 @@
+# Install Kata Containers on Debian
+
+These install instructions are based on the Ubuntu packages, but we need to install an updated version of `librdb1`.
```

Comment:
I'd love for us to offer packages for Debian, but I'd much rather we produce actual Debian packages rather than hoping the Ubuntu ones will (continue to!) work. This would require us to create Debian packages under OBS, for example under:

- https://download.opensuse.org/repositories/home:/katacontainers:/releases:/x86_64:/master/

@marcov, @jcvenegas - is this something we could look at doing?

Related, we would need Jenkins slaves running Debian to test the packages and these instructions.

/cc @chavafg.

Reply:
OBS has projects for Debian 9 (and others), so that's definitely possible. We could start by enabling this, and it should reuse the source file already in place for Ubuntu. 
There may be some things to update in the build recipes, but that's basically all it takes to get Debian builds :)

Reply:
I agree a proper Debian packaging would be great. I am more than happy to test changes.